streaming_jupyter_integrations/magics.py

```diff
@@ -623,17 +625,26 @@ def __retract_user_as_something_is_executing_in_background() -> None:
 
     @staticmethod
     def __read_global_config() -> Dict[str, Any]:
-        if "FLINK_HOME" in os.environ:
-            with open(os.path.join(os.environ["FLINK_HOME"], "conf", "flink-conf.yaml"), 'r') as stream:
-                try:
-                    parsed_yaml = yaml.safe_load(stream)
-                    return parsed_yaml
-                except yaml.YAMLError as exc:
-                    print(exc)
+        if "FLINK_CONF_DIR" in os.environ:
+            config_path = os.path.join(os.environ["FLINK_CONF_DIR"], "flink-conf.yaml")
+            return Integrations.__read_flink_conf_yaml(config_path)
+        elif "FLINK_HOME" in os.environ:
+            config_path = os.path.join(os.environ["FLINK_HOME"], "conf", "flink-conf.yaml")
+            return Integrations.__read_flink_conf_yaml(config_path)
         else:
-            print("FLINK_HOME environment variable is not set, reading flink-conf skipped")
+            print("Neither FLINK_CONF_DIR nor FLINK_HOME environment variable is set, reading flink-conf skipped.")
```

Comment:
... environment variable was set. Skipping reading from flink-conf.

Reply:
I think present simple is fine here.
Splitting into two sentences looks good.